extension/test/e2e/credit-card-3ds-native.spec.js

```diff
@@ -1,18 +1,15 @@
-const ctpClientBuilder = require('../../src/ctp')
-const { routes } = require('../../src/routes')
-const config = require('../../src/config/config')
-const httpUtils = require('../../src/utils')
+import ctpClientBuilder from '../../src/ctp.js'
+import { routes } from '../../src/routes.js'
+import config from '../../src/config/config.cjs'
+import httpUtils from '../../src/utils.js'
+import testUtils from './e2e-test-utils.js'
```

Comment:
```suggestion
import { assertPayment, createPayment, initPuppeteerBrowser, serveFile } from './e2e-test-utils.js'
```

Why don't you do this?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/929/commits/ea1042f8703bf272a62c2f21a8242284d743fdab

I changed it at all places, where I don't need the file as a object.